datacats/cli/create.py

```diff
@@ -20,7 +20,7 @@ def create(opts):
     """Create a new environment
 
 Usage:
-  datacats create [-bin] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
+  datacats create [-bn] [-i|--address=IP] [--ckan=CKAN_VERSION] ENVIRONMENT_DIR [PORT]
```

Comment:
I have to eat my words here. I guess it is possible for to set an address and still not start the containers.